src/vs/monaco.d.ts

```diff
@@ -3619,6 +3619,12 @@ declare namespace monaco.editor {
 		 * 'multiFile'  triggers occurrence highlighting across valid open documents
 		 */
 		occurrencesHighlight?: 'off' | 'singleFile' | 'multiFile';
+		/**
+		 * Controls delay for occurrences highlighting
+		 * Defaults to 250.
```

Comment:
Do we need a sane maximum?

Reply:
Right now it's set to 2k maximum, I think that's about all that any sane person would want. 2000 already feels comically slow, maybe 1k could be reasonable. I can't think of a reason to explicitly want to increase this delay.

Reply:
The minimum was called out, if we have a max that should be too is all